app/assets/javascripts/answers.js

```diff
@@ -43,17 +43,21 @@ $('#answer_text').keyup(enableSubmitAnswer);
       // prevents bootstrap from automatically adding a hidden attribute; we are explicitly hiding and showing using jquery instead
        ev.preventDefault(); 
        $("#answers").fadeIn();
+       $('#respond-no').fadeIn();
+       $('#respond-yes').fadeIn();
```

Comment:
FYI in future [you can also do this](https://api.jquery.com/multiple-selector/):
```
$("#answers, #respond-no, #respond-yes, #response").fadeIn();
```
Even better would be to give each of these a class, then search by class to make things fade in/out at certain times. That way new elements would just need to have that class; we wouldn't need to remember to explicitly fade in via the JS. Also, if elements are removed we won't have a failing line in the JS.